airflow/www/static/js/tree/InstanceTooltip.jsx

```diff
@@ -78,86 +76,27 @@ const InstanceTooltip = ({
     });
   }
 
-  const taskIdTitle = isGroup ? 'Task Group Id: ' : 'Task Id: ';
-
   return (
-    <Box fontSize="12px" py="4px">
+    <Box fontSize="12px" py="2px">
       {group.tooltip && (
         <Text>{group.tooltip}</Text>
       )}
       <Text>
-        <Text as="strong">Status:</Text>
+        {isGroup ? 'Overall ' : ''}
+        Status:
         {' '}
         {state || 'no status'}
       </Text>
-      {isGroup && (
-        <>
-          <br />
-          <Text as="strong">Group Summary</Text>
-          {groupSummary}
-        </>
-      )}
-      {group.isMapped && (
-        <>
-          <br />
-          <Text as="strong">
-            {mappedStates.length}
-            {' '}
-            {mappedStates.length === 1 ? 'Task ' : 'Tasks '}
-            Mapped
-          </Text>
-          {mapSummary}
-        </>
-      )}
-      <br />
-      <Text>
-        {taskIdTitle}
-        {taskId}
-      </Text>
-      <Text whiteSpace="nowrap">
-        Run Id:
-        {' '}
-        {runId}
-      </Text>
-      {operator && (
-      <Text>
-        Operator:
-        {' '}
-        {operator}
-      </Text>
-      )}
-      <Text>
-        Duration:
-        {' '}
-        {formatDuration(duration || getDuration(startDate, endDate))}
-      </Text>
-      <br />
-      <Text as="strong">UTC</Text>
-      <Text>
-        Started:
-        {' '}
-        {startDate && formatDateTime(moment.utc(startDate))}
-      </Text>
-      <Text>
-        Ended:
-        {' '}
-        {endDate && formatDateTime(moment.utc(endDate))}
-      </Text>
-      <br />
-      <Text as="strong">
-        Local:
-        {' '}
-        {moment().format('Z')}
-      </Text>
+      {isGroup && groupSummary}
       <Text>
         Started:
         {' '}
         {startDate && formatDateTime(startDate)}
       </Text>
       <Text>
-        Ended:
+        Duration:
         {' '}
-        {endDate && formatDateTime(endDate)}
+        {formatDuration(duration || getDuration(startDate, endDate))}
```

Comment:
What does this show when the task is still in progress?

Do we not show end date anymore?

Reply:
`endDate` is in the details but not in the tooltip.
When in progress it will use `Date.now()` instead of enddate